### ISSUE TITLE
Switch back to previous application after "code --wait"

### ISSUE THREAD

**Issue by stefanhaller** [microsoft/vscode#172574]
When using `code --wait` in, for example, Terminal.app on Mac, it would be convenient if it would switch back to Terminal.app after closing the file in VS Code.

The `subl` command-line tool of Sublime Text does this, and it's awesome. It has an option to disable this behavior ("-s or --stay: Keep the application activated after closing the file"); I suppose for `code` it could also be the other way round, with an opt-in parameter in order to not change existing behavior.

For now I'm approximating the behavior on Mac with a small wrapper script:
```
#/bin/sh

pid="$(osascript -e 'tell application "System Events" to get unix id of first process whose frontmost is true')"
code -w "$@"
osascript -e "tell application \"System Events\" to set frontmost of processes whose unix id is $pid to true"
```

**Comment by bpasero** [comment]
Yes, I would like this as well but never got to how to actually implement this across all platforms. If someone has suggestions, I am open to hear them.

**Comment by elia** [comment]
Not if it can help but TextMate does that too, this is a reference to the CLI accepting `--wait` https://github.com/textmate/textmate/blob/2624e59e2c51354a8282f7fdfbc9dff350bd8ca7/Applications/mate/src/mate.mm#L281
